docs/tutorials/rigetti/getting_started.ipynb

```diff
@@ -11,7 +11,7 @@
   },
```

Comment:
<p>Can this be replaced with <code>cirq.plot_state_histogram</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/5372/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
Done.